src/librustc_back/target/linux_musl_base.rs

```diff
@@ -1,71 +0,0 @@
-// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution and at
-// http://rust-lang.org/COPYRIGHT.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-
-use target::TargetOptions;
-
-pub fn opts() -> TargetOptions {
-    let mut base = super::linux_base::opts();
-
-    // Make sure that the linker/gcc really don't pull in anything, including
-    // default objects, libs, etc.
```

Comment:
Hm now that I see the diff it's actually pretty unfortunate that we're losing comments like this. I personally consider this knowledge critical in explaining why all these various arcane flags exist.

@brson do you have thoughts about this? JSON doesn't have comments, but we could perhaps vendor a TOML parser or something like that?